app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -221,8 +221,8 @@
           entries="claim.selectedLabels"
           key-placeholder="label"
           value-placeholder="value"
-          key-validator="[a-zA-Z][a-zA-Z0-9_-]*"
-          key-validator-error-tooltip="A valid label name is an alphanumeric (a-z and 0-9) string beginning with a letter that may contain underscores and dashes."
+          key-validator="[a-zA-Z][a-zA-Z0-9_.-\/]*"
```

Comment:
It looks like the label validation is more complicated than this, and the original regex on this page was incorrectly copied from an env variables editor. See

https://github.com/spadgett/origin-web-console/blob/57ae92d906931b0f60b6fa73833c9dc800f26fca/app/scripts/directives/labels.js#L38-L41

Can you try to use the `label-editor` directive here instead? Here's an example:

https://github.com/spadgett/origin-web-console/blob/57ae92d906931b0f60b6fa73833c9dc800f26fca/app/views/create-route.html#L21-L26

Reply:
But this one still does not do "/" - like kubernetes.io/test ?

Reply:
Do we want to update label regexp too?

Reply:
It should split on `/` before testing the regex if you look further down.

https://github.com/spadgett/origin-web-console/blob/57ae92d906931b0f60b6fa73833c9dc800f26fca/app/scripts/directives/labels.js#L75

Reply:
k, let me check what I can do :)

Reply:
k, having some trouble to use `label-editor` directive...  there is some styles clashing, which makes directive not visible (my assumption)... my UI skills are rusty :/

Reply:
when it comes to directives and angularjs - I have to say - pass :/

Reply:
@mjudeikis I hope you don't mind: I went ahead and opened #2423 to use the label-editor

Reply:
all good :) Thanks